### DISCUSSION TITLE
Imageviewer 为什么不基于 Image 组件开发？

### DISCUSSION THREAD

**Discussion by luckymore** [ant-design/ant-design-mobile#5716] [Q&A]
Image组件更容易控制 img 不是吗

**Comment by miracles1919** [comment]
大概是没用到 `Image` 提供的能力

**Reply by luckymore** [reply]
嗯呐，但是感觉应该趁早融合下，lazy、error、fit都是特别常用的基本功能。而且`ImageViewer`的`image`属性只能传 url，也不能自定义。
